05-workflow.Rmd

```diff
@@ -0,0 +1,537 @@
+# Workflow 
+
+## Why workflow? 
+
+> "I think of workflow as one of my "secret" powers: one of the reasons that I've been able to accomplish so much is that I devote time to analysing and improving my workflow. I highly encourage you to do the same!".....Hadley Wikham
+
+-   Workflow makes the process of writing Shiny apps more enjoyable, and helps your skills improve more quickly.
+
+## Learning objectives
+
+- The goal of this chapter is to help you improve three important Shiny workflows:
+
+
+    -   Learn basic **development cycle** for creating apps,making changes, and quickly expirement with the results.
+
+    -   Learn how to **debug** Shiny apps
+    
+<!-- your comment 
+    
+    Debugging, the workflow where you figure out what's gone wrong with your code and then brainstorm solutions to fix it.
+-->
+    
+    -   Learn how to write **self-contained reprexes**
+
+    
+<!-- your comment 
+Writing reprexes, self-contained chunks of code that illustrate a problem. Reprexes are a powerful debugging technique, and they are essential if you want to get help from someone else.
+ --> 
+
+       
+       
+  ![](images/workflow.png)
```

Comment:
Could you upload the figures, please.
But put the files in `./images/05-workflow/`
Then make sure that your image-links are rewritten to point to `./images/05-workflow/workflow.png` etc